### DISCUSSION TITLE
How to set up the lifeform Sensor?

### DISCUSSION THREAD

**Discussion by Artemvar41** [BEEmod/BEE2-items#4090] [Tech Support / Q&A] [answered]
The editor thinks the lifeform sensor is the output of the signal, not the input. The question is how to make it so that when the player passes through the sensor of lifeforms, for example, the laser turns on?

**Comment by quat1024** [comment, marked as answer]
Due to puzzlemaker bugs fizzler items don't support outputs. Instead, use the ["Fizzler Output Relay"](https://github.com/BEEmod/BEE2-items/wiki/Fizzler-Output-Relay) item and place it in the same space as the fizzler. Connect your laser to the output relay instead of directly to the fizzler.

**Reply by Pareq** [reply]
it doesn't work multiple times how can i turn it on again
